dll/cpl/desk/desk.h

```diff
@@ -81,24 +81,42 @@ ULONG __cdecl DbgPrint(PCCH Format,...);
 #define MAX_DESK_PAGES        32
 #define NUM_SPECTRUM_BITMAPS  3
 
-/* This number must match DesktopIcons array size */
-#define NUM_DESKTOP_ICONS  4
+#define NUM_DESKTOP_ICONS  4  /* DesktopIcons array size */
+#define NUM_CHANGE_ICONS   5  /* IconChange array size */
```

Comment:
What's the difference between the two?

Reply:
First is for show/hide checkboxes, second is for the icons in listview.